FriendsOfPHP/PHP-CS-Fixer
# PhpUnitTestCaseStaticMethodCallsFixer - prepare for PHPUnit 8

To verify this run this:

```
rm composer.lock
rm -fr vendor
composer remove --dev --no-update php-cs-fixer/phpunit-constraint-isidenticalstring php-cs-fixer/phpunit-constraint-xmlmatchesxsd phpunitgoodpractices/traits
composer require --dev phpunit/phpunit:^8
PHP_CS_FIXER_FUTURE_MODE=1 php php-cs-fixer fix --rules=void_return tests/Fixer/PhpUnit/PhpUnitTestCaseStaticMethodCallsFixerTest.php
PHP_CS_FIXER_FUTURE_MODE=1 php php-cs-fixer fix --rules=void_return tests/Test/AbstractFixerTestCase.php
vendor/bin/phpunit tests/Fixer/PhpUnit/PhpUnitTestCaseStaticMethodCallsFixerTest.php
```

on branch `2.12` to see the failing test and then on these changes to see it passing.
